cargo-espflash/src/package_metadata.rs

```diff
@@ -84,17 +69,10 @@ impl PackageMetadata {
                 Some(meta) if meta.is_table() => {
                     let meta = meta.as_table().unwrap();
 
-                    espflash_meta.bootloader = meta
-                        .get("bootloader")
-                        .map(|bl| package.root().join(bl.as_str().unwrap()));
-
+                    // TO BE REMOVED?
```

Comment:
Could you remind me again what this is? I don't use cargo-espflash very often 😅.

Reply:
You could set the `partition-table`, `bootloader` and `format` in the package metadata, see https://github.com/esp-rs/espflash/tree/main/cargo-espflash#package-metadata. At the moment, I removed `baudrate` and `partition-table` properties. IMHO, we can also remove the format

Reply:
Ah, I see, thanks! Yes lets remove format too.